cli/src/org/partiql/shell/Shell.kt

```diff
@@ -261,6 +269,8 @@ class Shell(
             }
         }
     }
+
+    class ShellConfiguration(val isMonochrome: Boolean = false)
```

Comment:
Could also document what this `ShellConfiguration` class is for and document why certain attributes are included here rather than in the `Shell` constructor.

Reply:
The general thinking is, if the option/configuration comes in as user input, then it belongs in the ShellConfiguration. The ShellConfiguration could optionally be dumped into `$HOME/.partiql/shell.ion` or something of the likes. Then we deserialize that config to the object.

This allows use to have re-usable configurations as the shell is expanded

Reply:
So should some of the other REPL/Shell options be included in this configuration? The `-e`/`environment` option specifying the initial global environment would seem to belong in the `ShellConfiguration`. Similarly for `-p`/`--permissive` specifying if the evaluator should be run in permissive typing mode.